.github/workflows/ci-pulsarbot.yaml

```
@@ -34,6 +34,233 @@ jobs:
     steps:
       - name: Execute pulsarbot command
         id: pulsarbot
-        env:
+        uses: actions/github-script@v7
+        with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

warning in https://github.com/liangyepianzhou/pulsar/actions/runs/19063580190/job/54448699453#step:2:1